library/src/main/java/com/bumptech/glide/manager/RequestManagerRetriever.java

```diff
@@ -133,6 +149,136 @@ public RequestManager get(Fragment fragment) {
     }
   }
 
+  public RequestManager get(View view) {
+    if (view.getContext() == null) {
+      throw new IllegalArgumentException("Unable to obtain a request manager for a view without"
```

Comment:
```java
throw new IllegalArgumentException(
    "Unable to obtain a request manager for a view without a Context");
```

would help readabilty, and full-text-search for the message; and it should still fit within margin.